crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
What do you thin about the docs mention?

And I just realized, I need to push a fix to the spec. we do not have coverage over the function returned from `value`. I deleted it and forgot to replace it with an appropriate test.

How embarrassing for me 😲